src/code/dialogs/zoneDialog.js

```diff
@@ -97,18 +128,165 @@ const ZoneDialog = WDialog.extend({
           getSelectedOperation().removeZone(z.id);
         });
       }
+      if (z.points.length == 0) {
+        const addPoints = L.DomUtil.create("a", null, commandcell);
+        addPoints.textContent = " 🖍";
+        addPoints.href = "#";
+        L.DomEvent.on(addPoints, "click", (ev) => {
+          L.DomEvent.stop(ev);
+          this.ZonedrawHandler.zoneID = z.id;
+          this.ZonedrawHandler.enable();
+        });
+      } else {
+        const delPoints = L.DomUtil.create("a", null, commandcell);
+        delPoints.textContent = " 🧽";
+        delPoints.href = "#";
+        L.DomEvent.on(delPoints, "click", (ev) => {
+          L.DomEvent.stop(ev);
+          getSelectedOperation().removeZonePoints(z.id);
+        });
+      }
+      if (this.ZonedrawHandler && this.ZonedrawHandler._enabled) {
```

Comment:
This isn't triggering. Why isn't this.ZonedrawHandler present by the second draw of the dialog?